model/src/main/java/org/mskcc/cmo/metadb/model/web/PublishedMetadbSample.java

```diff
@@ -326,6 +339,14 @@ public void setSampleAliases(List<SampleAlias> sampleAliases) {
         return sampleAliases;
     }
 
+    public MetadbPatient getPatient() {
```

Comment:
actually I realize another thing - `patientAliases` only should be added to the published sample

Reply:
another thing regarding `patientAliases` - we should add `@JsonIgnore` to the Patient Metadata // where is that coming from?

Reply:
this is a screen shot of what the mocked published data looks like / what we should have the web service return for the `sampleAliases` and `patientAliases` contents

![image](https://user-images.githubusercontent.com/15623749/146829795-11e43d88-67c9-4d41-ae10-45d80c482f76.png)